src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -237,10 +237,18 @@ public static void GetPathRoot_Unix()
             Assert.False(Path.IsPathRooted(uncPath));
             Assert.Equal(string.Empty, Path.GetPathRoot(uncPath));
         }
-        
+
+        [Fact]
+        public static void IsPathRooted()
+        {
+            Assert.False(Path.IsPathRooted(null));
+        }
+
         // Testing invalid drive letters !(a-zA-Z)
         [PlatformSpecific(TestPlatforms.Windows)]
         [Theory]
+        [InlineData("")]
+        [InlineData(" ")]
```

Comment:
Absolutely. As this PR now contains little to nothing I will hold it for now and add some more test cases later today.